README.md

```diff
@@ -55,16 +55,16 @@ The Storefront is designed as an open source mobile Progressive Web Application
 
 ### Prerequisites
 
-Ensure that the following software are installed :
+Ensure that the following software are installed:
 
 * [Git](https://git-scm.com/downloads)
 * [Node.js](https://nodejs.org/en/download/)
 * [Visual Studio Code](https://code.visualstudio.com/) with the following extensions:<br/>
     * [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)<br/>
     * [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)<br/>
-* [Java JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/java-archive-javase8-2177648.html)(for unit testing, optional)
-* [Maven 3.5.2](https://archive.apache.org/dist/maven/maven-3/3.5.2/binaries/)(for unit testing, optional)
-* [IntelliJ IDEA](https://www.jetbrains.com/idea/)(for unit testing, optional)
+* [Java JDK 8] (http://www.oracle.com/technetwork/java/javase/downloads/java-archive-javase8-2177648.html)(for unit testing, optional)
```

Comment:
space will break the link, space should be before this (for unit testing, optional)